files/fare_transfer_rules_ft.md

```diff
@@ -13,8 +13,8 @@ File MUST contain the following attributes:
 
 Required Attributes	| Description										
 ----------			| -------------		
-`from_fare_class`	| An ID that identifies the `fare_class` that the passenger is coming from.  
-`to_fare_class`		| An ID that identifies the `fare_class` that the passenger is going to.  
+`from_fare_period`	| An ID that identifies the `fare_period` that the passenger is coming from.  
```

Comment:
Hmm... The fares are now specified by fare_period so I think it should be fare_period?  But it does get a bit confusing... like would we need bart_pm to muni_pm and muni_ev if it's possible that the transfer could cross time periods?  :(